packages/envapt/tests/.env.043-semantic-converters

```
@@ -0,0 +1,24 @@
+# port fixtures
+PORT_VALID=8080
+PORT_ZERO=0
+PORT_MAX=65535
+PORT_HIGH=65536
+PORT_NEG=-1
+PORT_FLOAT=3000.5
+PORT_JUNK=8080abc
+PORT_WS="   "
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PORT_WS key should go before the PORT_ZERO key

(UnorderedKey)